drivers/led_strip/ws2812_spi.c

```diff
@@ -67,20 +67,6 @@ static inline void ws2812_spi_ser(uint8_t buf[8], uint8_t color,
 	}
 }
 
-/*
- * Returns true if and only if cfg->px_buf is big enough to convert
- * num_pixels RGB color values into SPI frames.
- */
-static inline bool num_pixels_ok(const struct ws2812_spi_cfg *cfg,
-				 size_t num_pixels)
-{
-	size_t nbytes;
-	bool overflow;
-
-	overflow = size_mul_overflow(num_pixels, cfg->num_colors * 8, &nbytes);
-	return !overflow && (nbytes <= cfg->px_buf_size);
-}
-
```

Comment:
You should keep this function alive until `drivers: led_strip: Check length before updating LED strip`.

Reply:
Fixed